bin/oneoff/backfill_data/backfill_level_references.rb

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env ruby
+
+# https://codedotorg.atlassian.net/browse/TEACH-935
+# Context
+# Reference guides for curriculum was migrated to a framework that supports versioning
+# But, the corresponding change needed to update version for linked reference guides when
+# cloning scripts is currently missing. That is tracked by task TEACH-
+
+# This script is a backfill to fix the reference guide version for scripts that are already
+# cloned for 2024.
+
+require_relative '../../../dashboard/config/environment'
+
+# Override for the dry run variable using an commandline argument
+DRY_RUN = !((ARGV.find {|arg| arg.casecmp('-dryrun')}).nil?)
+
+def update_reference_guide_format(unit_group_name, course_version_id, reference_guide_link)
+  # Reference guide formats
+  # in the old unversioned framework - /docs/concepts/patterns/random-list-access/
+  # in the versioned framework - /courses/csd-2024/guides/animation-tab
+  if !reference_guide_link.starts_with?("/docs/") && !reference_guide_link.starts_with?("/courses/")
```

Comment:
Update this check to look for /docs/concepts instead of just docs as we have some static pages of docs which are hosted under /docs.

Reply:
https://codedotorg.slack.com/archives/C045UAX4WKH/p1712254216725949?thread_ts=1711661624.079219&cid=C045UAX4WKH